pstore/config/tsdb/tsdb.go

```diff
@@ -27,9 +21,27 @@ const (
 	contentTypeJSON = "application/json"
 )
 
+var (
+	supportedTypes = map[types.Type]bool{
+		types.Bool:       true,
+		types.Int8:       true,
+		types.Int16:      true,
+		types.Int32:      true,
+		types.Int64:      true,
+		types.Uint8:      true,
+		types.Uint16:     true,
+		types.Uint32:     true,
+		types.Uint64:     true,
+		types.Float32:    true,
+		types.Float64:    true,
+		types.GoTime:     true,
+		types.GoDuration: true,
+		types.String:     true,
+	}
+)
```

Comment:
FYI: if the values will only ever be true (i.e. you just want a set), then you could make the value type struct{} and just depend on presence.